code/modules/urist/items/uristweapons.dm

```diff
@@ -115,4 +115,95 @@ Please keep it tidy, by which I mean put comments describing the item before the
 	edge = 0
 	lock_picking_level = 2
 
-///obj/item/weapon/gun/launcher/crossbow/bow //gonna mess with this
\ No newline at end of file
+///obj/item/weapon/gun/launcher/crossbow/bow //gonna mess with this
+
+
+//RS Weapons
+
+
+
+/obj/item/weapon/BronzeDagger
+	name = "Bronze Dagger"
+	desc = "Short but pointy."
+	icon = 'icons/urist/items/uristweapons.dmi'
+	icon_state = "bronze dagger"
+	item_state = "knife"
+	force = 12
+	throwforce = 5
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	attack_verb = list("stabbed", "slashed", "sliced", "cut")
+	w_class = 2.0
+	sharp = 1
+	edge = 1
+
+
+/obj/item/weapon/AddyDagger
+	name = "Adamantite Dagger"
+	desc = "Short but pointy."
+	icon = 'icons/urist/items/uristweapons.dmi'
+	icon_state = "green dagger"
+	item_state = "knife"
+	force = 20
+	throwforce = 9
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	attack_verb = list("stabbed", "slashed", "sliced", "cut")
+	w_class = 2.0
+	sharp = 1
+	edge = 1
+
+
+/obj/item/weapon/RuneDagger
+	name = "Runite Dagger"
+	desc = "Short but pointy."
+	icon = 'icons/urist/items/uristweapons.dmi'
+	icon_state = "blue dagger"
+	item_state = "knife"
+	force = 27
+	throwforce = 15
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	attack_verb = list("stabbed", "slashed", "sliced", "cut")
+	w_class = 2.0
+	sharp = 1
+	edge = 1
+
+/obj/item/weapon/urist/BronzeSword
+	name = "Bronze Sword"
+	desc = "A razor sharp sword."
+	icon = 'icons/urist/items/uristweapons.dmi'
+	icon_state = "bronze sword"
+	item_state = "claymore"
+	force = 15
+	throwforce = 9
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	attack_verb = list("attacked", "slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut")
+	w_class = 2.0
```

Comment:
swords should be w_class = 3 or 4. 1 is behind ears, 2 is pockets, 3 is backpacks and anything higher can't be fit.  I'd say 4 for swords, because that fits with the existing swords.